.github/workflows/Whitespace.yml

```diff
@@ -0,0 +1,23 @@
+name: Whitespace
+
+permissions: {}
```

Comment:
We need `contents: read` to be able to clone the repo, right?

Reply:
Doesn't seem to be necessary since [this is working](https://github.com/JuliaLang/julia/actions/runs/10132458011/job/28016444891?pr=55281), and I copied this from https://github.com/JuliaLang/julia/blob/197295c84aab217bffde01a4339f1d0e8e95fb98/.github/workflows/Typos.yml#L3 (as you probably figured from https://github.com/JuliaLang/julia/pull/55281#discussion_r1694254720). Side note, you seem to have made [the same comment before](https://github.com/JuliaLang/julia/pull/51704#discussion_r1359478966)